### DISCUSSION TITLE
possibility to type exact value of numeric goals

### DISCUSSION THREAD

**Discussion by eastblue056** [iSoron/uhabits#1188] [Feature Requests]
I use numeric goals to track how many minutes I listens to audio books for example and it would be nice if I could type in the exact value instead of having to scroll to enter it.

**Comment by hiqua** [comment]
I think this is present in the latest `dev` version. But IMO we're still missing something: having a dedicated data model for time, where we could easily record the time spent on something, and adapt the goal as well.

**Comment by ke352802081770314** [comment]
In 2.0.2 I find that if I tap on the scroller wheel, a numerical keyboard appears below and I can enter the number. Sometimes it even appears straight away, and sometimes not, I haven't figured out yet what the condition is. Using the wheel is something I avoid because I find it very inconvenient.

**Reply by hiqua** [reply]
> Sometimes it even appears straight away, and sometimes not, I haven't figured out yet what the condition is.

It should appear if you long-press the number. This will be improved in the next release.
